tests/unit/components/search/SearchForm.spec.js

```diff
@@ -138,16 +143,31 @@ describe('components/search/SearchForm', () => {
 
     context('with a selected entity suggestion', () => {
       it('routes to the entity page', async() => {
-        const wrapper = factory();
+        const state = {
+          active: true,
+          userParams: {
+            query: ''
+          },
+          view: 'grid'
+        };
+        const wrapper = factory({ store: store(state) });
 
         wrapper.setData({
           suggestions: parsedSuggestions,
```

Comment:
this doesn't actually need to be set if it's setting the value for 'selectedSuggestion' already below.